src/tests/mock/map/mock_cmapclient.h

```diff
@@ -9,7 +9,13 @@ using ::testing::Invoke;
 
 class CMapClient_Mock : public CMapClient {
     public:
-        MOCK_CONST_METHOD1(IsNearby, bool(const RoseCommon::CRoseClient*));
-        MOCK_METHOD0(canBeDeleted, void(void));
-        MOCK_METHOD1(Send, bool(RoseCommon::CRosePacket&));
+      CMapClient_Mock() : CMapClient() {};
+      CMapClient_Mock(Core::INetwork* _sock) : CMapClient() {
```

Comment:
The two constructors here aren't needed as there's a default CMapClient() and the CMapClient code is never called.